xz/FeatureModel.xml

```diff
@@ -2073,13 +2073,13 @@
     <constraint>!BytesDecompr | !NoBytesDecompr</constraint>
     <constraint>PercentDecompr | NoPercentDecompr</constraint>
     <constraint>!PercentDecompr | !NoPercentDecompr</constraint>
-    <constraint>Blocks =&gt; xz</constraint>
-    <constraint>lzma =&gt; LZMA1</constraint>
-    <constraint>xz =&gt; !LZMA1</constraint>
-    <constraint>FlushTO =&gt; LZMA2 | Delta</constraint>
-    <constraint>Test =&gt; KeepIn</constraint>
-    <constraint>TestDecompr =&gt; Test | Decompr</constraint>
-    <constraint>Memory =&gt; MaxMemCompr | MaxMemDecompr</constraint>
+    <constraint>Blocks | !xz</constraint>
```

Comment:
Wouldn't $$Blocks \implies xz$$ be equivalent to $$!Blocks \mid xz$$ instead of $$Blocks \mid !xz$$ (and similar for all the others) since

$$ P \implies Q\  \equiv\   !P \mid Q $$

holds?

Reply:
Ohhhhh, I see where the problem is here. I would have to ask @Keller-1998 . 
I interpreted that as an inequation instead of an implication.

Because interpreting it as an inequation would be like that:
$Blocks >= xz$ is equivalent to $xz ⟹ Blocks$ when $xz$ and $Blocks$ have only the values $0$ or $1$?
But if @Keller-1998 meant the implication (which is also not supported by our format, unfortunately), then reasoning would be correct, @bnico99 .
This can be easily changed, but we need some input from @Keller-1998 .

Reply:
@bnico99 was right here, these were meant to be implications.